build_web_compilers/lib/src/dart_source_cleanup.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:build/build.dart';
+
```

Comment:
nit: add a small comment here `deletes all dart files it finds if its enabled`.

It might actually be worth creating a `FileDeleterBuilder` which takes the input extensions in the constructor, and then we can easily just add a factory method for each one we want to add? This would probably live in the `build` package in that case but its such a common thing to want to do I think that would make sense.

Reply:
Agree on making a `FileDeletingBuilder`. Will take a look at that.